src/fparser/readfortran.py

```diff
@@ -348,18 +388,28 @@ def __init__(self, comment, linenospan, reader):
         self.comment = comment
         self.span = linenospan
         self.reader = reader
+
     def __repr__(self):
         return self.__class__.__name__+'(%r,%s)' \
                % (self.comment, self.span)
+
     def isempty(self, ignore_comments=False):
-        return ignore_comments # or len(self.comment)<2
+        '''
+        Returns true if there is no comment. This is always the case when
+        ignoring comments.
+        '''
+        if ignore_comments:
+            return True
+        else:
+            return self.comment.strip() == ''
```

Comment:
I can't remember if I found a problem. I tried reverting the change and the unit tests still pass, so I guess not.

I think I just wanted the function to do what it seemed to be claiming to do rather than returning an affirmation of comment just because you didn't want to ignore comments.

How do you want to handle this?

Reply:
Would you mind reverting the change as #71 deals with this? Thanks.